veil_falls.logic

```diff
@@ -1,2 +1,77 @@
 area VeilFalls:
-  room Main
+  room Main:
+    node BottomWater:
+      exit BottomWaterFallFusion
+    node BottomWaterfall:
+      entrance BottomWaterfallFusion
+    BottomWater -> BottomWaterFall: flag VeilFallsBottomWaterfallFusion
+    node Bottom:
+      chest BottomHP
+      door SouthEast
+      door SouthWest
+    Bottom <-> BottomWater: item Flippers
+    Bottom <-> BottomEast: (item Cape | Item Flippers)
+    node BottomEast:
+      chest Rupee1
+      chest Rupee2
+      chest Rupee3
+      enemy LikeLike20
+      exit BottomEastDigCave
+    room BottomEastDigCave:
+      entrance BottomEastDigCave
+    BottomEast -> BottomEastDigCave: item MoleMitts
```

Comment:
```suggestion
    BottomEast -> BottomEastDigCave: item MoleMitts
    BottomEastDigCave -> BottomEast
```

Reply:
Is that neccesary when the exit part of the door is in BottomEast?

Reply:
It's not, you're right. My bad